pkg/nuctl/command/deploy.go

```diff
@@ -191,3 +194,33 @@ func parseResourceAllocations(values stringSliceFlag, resources *v1.ResourceList
 
 	return nil
 }
+
+func parseVolumes(volumes stringSliceFlag) []functionconfig.Volume {
```

Comment:
Please add a unit test for this function in `deploy_test.go` (sync to latest, it's there)